test/utils/AggregationUtil.test.js

```diff
@@ -555,4 +545,51 @@ describe('AggregationUtil', () => {
       expect(aggregationUtil.dataUtil.activeEndpoints).to.eql(aggregationUtil.initialActiveEndpoints);
     });
   });
+
+  describe('filterByActiveRange', () => {
+    const mockResultsArg = [
+      { key: '2025-08-04', value: { dataList: [] } },
+      { key: '2025-08-05', value: { dataList: [] } },
+      { key: '2025-08-06', value: { dataList: [] } },
+      { key: '2025-08-07', value: { dataList: [] } },
+      { key: '2025-08-08', value: { dataList: [] } },
+      { key: '2025-08-09', value: { dataList: [] } },
+    ];
+
+    beforeEach(() => {
+      aggregationUtil = createAggregationUtil(data, {
+        ...defaultOpts,
+        timePrefs: { timezoneName: 'US/Eastern' },
+      });
+    });
+
+    it('does NOT include data from the last date when endpoint range ends at midnight', () => {
+      aggregationUtil.initialActiveEndpoints.range = [
+        1754366400000, // 2025-05-05 at 00:00 (midnight) in US/Eastern
+        1754625600000, // 2025-05-08 at 00:00 (midnight) in US/Eastern
```

Comment:
```suggestion
        1754366400000, // 2025-08-05 at 00:00 (midnight) in US/Eastern
        1754625600000, // 2025-08-08 at 00:00 (midnight) in US/Eastern
```
Super small nit - months are incorrect in the comments :)